api/envoy/api/v2/route/route_components.proto

```diff
@@ -1010,6 +1010,17 @@ message RetryPolicy {
 
   // HTTP headers which must be present in the request for retries to be attempted.
   repeated HeaderMatcher retriable_request_headers = 10;
+
+  // The name of the retry policy. The field is used for pluggable retry policy.
+  string name = 11 [(validate.rules).string = {min_bytes: 1}];
```

Comment:
We can delete this and just use type_url mapped loading for new extensions (as per the work that @kyessenov has been doing)

Reply:
I removed the name filed but the compilation failed. 
```c
bazel-out/k8-fastbuild/bin/source/common/config/_virtual_includes/utility_lib/common/config/utility.h:240:63: error: no member named 'name' in 
'envoy::config::route::v3::RetryPolicy'
    return Utility::getAndCheckFactoryByName<Factory>(message.name());
```

The method requires the name of the message to fetch the factory.

Reply:
Done. Removed `name` field.